### PR TITLE
update dropdown tick and tooltip demo to use the small icons

### PR DESCRIPTION
**Pull request by msendlakowski**
## Description
Supporting 4 small icons on DS4 (which are used here or in ebayui-core):
* cart
* confirmation
* information
* tick

Updated the tick in the dropdown mixin to use the small icon.
Updated the information icon in the tooltip docs to use the small icon.

## Context
Minor UI effect, but definitely noticeable on the tooltip.

### Large icons (before)
Screen Shot 2019-08-21 at 2 51 57 PM
Screen Shot 2019-08-22 at 1 34 50 PM

### Small icons (after)
Screen Shot 2019-08-21 at 2 50 28 PM
Screen Shot 2019-08-22 at 1 32 21 PM

### DS4 small icon mapping:
Screen Shot 2019-08-22 at 1 26 44 PM

### CODE REVIEW

**Comment by msendlakowski** [comment]
> I thought we were going to do the alias route where small maps to medium for ds4

I thought we decided we wouldn't support the small icons on ds4.  I'm not sure how the aliasing would work either. Would we create an `icon-small` set of less files for ds4 and strip out the "-small" for the mixin names?  That's going to be a lot of duplication in the CSS.

**Comment by agliga** [comment]
> > I thought we were going to do the alias route where small maps to medium for ds4
> 
> I thought we decided we wouldn't support the small icons on ds4. I'm not sure how the aliasing would work either. Would we create an `icon-small` set of less files for ds4 and strip out the "-small" for the mixin names? That's going to be a lot of duplication in the CSS.

My understanding is we would basically have another style in the CSS class like so:
For background
```
.icon-tick,
.icon-tick-small {
    // Mixin with icon-tick
}
```
Foreground we would need just to add a new SVG with a new id, however I don't think that might be as necessary.

What I remember the discussion was, was we would only implement the icons which we would need to remap, not all small icons.

**Comment by seangates** [comment]
Duplication should be okay, as that effort addresses one of our core values: developer experience. Let's make it very, very easy for developers and remove some choices for them (e.g. let's avoid having them write logic for their icons).

And, as @agliga stated above, we only need to do this for a handful of icons, not all the icons.

File: src/svg/ds4/icons.svg [hunk 1]
```diff
@@ -146,6 +155,9 @@
     <symbol id="icon-information" viewBox="0 0 24 24">
         <path fill="#0654ba" d="M12.007 9.753a.849.849 0 0 0-.849.849v6.553a.85.85 0 0 0 1.683.004v-6.557a.849.849 0 0 0-.834-.849zm1.057-2.88a1.016 1.016 0 1 1-2.032 0 1.016 1.016 0 0 1 2.032 0zm-1.057 17.141C5.376 24.014 0 18.638 0 12.007 0 5.376 5.376 0 12.007 0c6.631 0 12.007 5.376 12.007 12.007v.015c-.008 6.625-5.381 11.992-12.007 11.992zm0-22.261c-5.671 0-10.268 4.597-10.268 10.268s4.597 10.268 10.268 10.268 10.268-4.597 10.268-10.268S17.678 1.753 12.007 1.753z"/>
     </symbol>
+    <symbol id="icon-information-small" viewBox="0 0 24 24">
```
Comment (agliga) [inline src/svg/ds4/icons.svg]: In order for this to be in sync with my version call this `icon-information-filled-small`

File: src/svg/ds4/icons.svg [hunk 2]
```diff
@@ -109,6 +115,9 @@
     <symbol id="icon-confirmation" viewBox="0 0 24 24">
         <path fill="#5ba71b" d="M12.007 24.014C5.376 24.014 0 18.638 0 12.007 0 5.376 5.376 0 12.007 0c6.631 0 12.007 5.376 12.007 12.007v.015c-.008 6.625-5.381 11.992-12.007 11.992zm0-22.261c-5.671 0-10.268 4.597-10.268 10.268s4.597 10.268 10.268 10.268 10.268-4.597 10.268-10.268S17.678 1.753 12.007 1.753zM11.13 17.6h-.003a.736.736 0 0 1-.485-.182L5.968 13.44a.74.74 0 1 1 .961-1.125l4.076 3.477 6.136-8.056a.744.744 0 1 1 1.181.906l-6.607 8.68a.722.722 0 0 1-.512.292h-.087l.014-.014z"/>
     </symbol>
+    <symbol id="icon-confirmation-small" viewBox="0 0 24 24">
```
Comment (agliga) [inline src/svg/ds4/icons.svg]: Rename to `icon-confirmation-filled-small`

File: src/svg/ds4/icons.svg (hunk 1 above)
Comment (msendlakowski) [reply src/svg/ds4/icons.svg]: As discussed offline, I won't update this since for DS4 it's only used in docs and demos on skin or ebayui.

**Pull request merged**